helm-charts/FATE/values-template-example.yaml

```diff
@@ -120,26 +120,28 @@ backend: eggroll
   # accessMode: ReadWriteMany
   # size: 1Gi
   # spark: 
-    # master: spark://spark-master:7077
-    # home: 
     # cores_per_node: 20
     # nodes: 2
+    # existingSpark: spark://192.168.99.1:7077
+    # driverHost: fateflow
+    # driverHostType: NodePort
+    # portMaxRetries: 16
+    # driverStartPort: 30997
+    # blockManagerStartPort: 31107
+    # pysparkPython: 
   # hdfs:
-    # name_node: hdfs://namenode:9000
+    # existingHdfs: hdfs://namenode:9000
```

Comment:
why not use `name_node`?

Reply:
You only need to set it when you use an existing HDFS.